packages/ng/file-upload/file-entry/file-entry.component.html

```diff
@@ -0,0 +1,110 @@
+<dl class="fileEntry" [class.mod-media]="media()" [ngClass]="dlClasses()">
+	@if (!media()) {
```

Comment:
could we fix (in another PR) the prettier parser issue?

We should add something like this in prettier config

```json
"overrides": [
    {
      "files": ["*.component.html", "*.page.html"],
      "options": {
        "parser": "angular"
      }
    }
  ]
```